src/pieces/autoSpookyness.ts

```diff
@@ -0,0 +1,36 @@
+import { GUILDS } from '@root/config';
+import { Client } from 'discord.js';
+import { schedule } from 'node-cron';
+import { SpookyError } from '@lib/types/errors';
+
+async function register(bot: Client): Promise<void> {
+	// Runs of the first of October at midnight
+	schedule('0 0 1 10 *', () => {
+		enableSpook(bot)
+			.catch(async error => bot.emit('error', error));
+	});
+
+	// Runs on the first of November at midnight
+	schedule('0 0 1 11 *', () => {
+		disableSpook(bot)
+			.catch(async error => bot.emit('error', error));
+	});
+}
+
+async function enableSpook(bot:Client) {
+	const guild = bot.guilds.cache.get(GUILDS.MAIN);
+
+	await guild.setIcon(`${__dirname}/../../../assets/images/spoopylogo.png`, 'Spoopifying the server.');
```

Comment:
I know this is how they do it everywhere else but I still cry every time I see it